instauto/api/actions/profile.py

```diff
@@ -43,9 +43,9 @@ def profile_update(self, obj: Update):
         self._profile_act(obj)
 
     def profile_info(self, obj: Info) -> Union[Dict, int]:
-        if obj.user_id is None:
+        if obj.user_id is None and obj.username is None:
```

Comment:
The original method used the users own id if none was provided - which ive kept in, so the request should still run fine. This could be useful if someone wants an easy way to get their own userinfo without having to grab their userid first, but I can still change it to raise an exception if you want?

Reply:
I think the user of `instauto` would not expect an empty struct to be filled with the client's account. Or it should be like that everywhere, but that is currently not the case.

You can get your own user id with `self.state.user_id`